package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "proj-happycows-s23-6pm-3",
+  "version": "1.0.0",
+  "description": "# Deployments",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
Why is this here?   My guess is that someone did an npm install in the wrong directory and created this by accident.   

We only want a package.json inside the frontend directory.

Can we remove this from the PR?